MaxMind.GeoIP2.UnitTests/WebServiceClientTests.cs

```diff
@@ -36,43 +36,36 @@ public WebServiceClientTests()
         public static readonly object[][] TestCases =
         {
 #pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-            new object[] {"country", (ClientRunner) (async (c, i) => c.Country(i)), typeof(CountryResponse)},
+            ["country", (ClientRunner) (async (c, i) => c.Country(i)), typeof(CountryResponse)],
 #pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-            new object[] {"city", (ClientRunner) (async (c, i) => c.City(i)), typeof(CityResponse)},
+            ["city", (ClientRunner) (async (c, i) => c.City(i)), typeof(CityResponse)],
 #pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-            new object[] {"insights", (ClientRunner) (async (c, i) => c.Insights(i)), typeof(InsightsResponse)},
+            ["insights", (ClientRunner) (async (c, i) => c.Insights(i)), typeof(InsightsResponse)],
 #pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
-            new object[]
-                {"countryAsync", (ClientRunner) (async (c, i) => await c.CountryAsync(i)), typeof(CountryResponse)},
-            new object[] {"cityAsync", (ClientRunner) (async (c, i) => await c.CityAsync(i)), typeof(CityResponse)},
-            new object[]
-                {"insightsAsync", (ClientRunner) (async (c, i) => await c.InsightsAsync(i)), typeof(InsightsResponse)}
+            ["countryAsync", (ClientRunner) (async (c, i) => await c.CountryAsync(i)), typeof(CountryResponse)],
+            ["cityAsync", (ClientRunner) (async (c, i) => await c.CityAsync(i)), typeof(CityResponse)], ["insightsAsync", (ClientRunner) (async (c, i) => await c.InsightsAsync(i)), typeof(InsightsResponse)
```

Comment:
Should the insightsAsync part be on its own line?

Reply:
Thanks, I did that.